Xcode/Sources/HttpResponse.swift

```diff
@@ -158,7 +161,7 @@ public enum HttpResponse {
 
     func content() -> (length: Int, write: ((HttpResponseBodyWriter) throws -> Void)?) {
         switch self {
-        case .ok(let body)             : return body.content()
+        case .ok(let body, _)             : return body.content()
```

Comment:
Please remove spaces before the `:` to keep it in line with the lines below.

Reply:
Done